magicctapipe/scripts/lst1_magic/DL1_to_DL2.py

```diff
@@ -0,0 +1,166 @@
+"""
+This script creates the bashscripts necessary to apply "lst1_magic_dl1_stereo_to_dl2.py"
+to the DL1 stereo data (real and MC). It also creates new subdirectories associated with
+the data level 2. The DL2 files are saved at:
+WorkingDirectory/DL2/
+and in the subdirectories therein.
+
+Usage:
+$ python DL1_to_DL2.py
+
+"""
+
+import os
+import numpy as np
+import glob
+import yaml
+import logging
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.StreamHandler())
+logger.setLevel(logging.INFO)
+
+def DL1_to_2(target_dir):
```

Comment:
please call it `DL1_to_DL2`, no reason to have it shortened

Reply:
Done.